### PR TITLE
fix(curriculum): typo in Decimal to Binary Converter - Step 26

### PR DESCRIPTION
**Pull request by RGHANILOO**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

typo on the last paragraph changed from 'input' to 'quotient'

### CODE REVIEW

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/644b71f678b5e1687666c88d.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ dashedName: step-26
 
 Like you saw in the last step, division can lead to a floating point number, or a number with a decimal point. The best way to handle this is to round down to the nearest whole number.
 
-Use the `Math.floor()` function to round down the quotient of `input` divided by `2` before it's assigned to `input`.
```
Comment (RGHANILOO) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/644b71f678b5e1687666c88d.md:12]: 'input' is the incorrect assignment

**Review by huyenltnguyen — APPROVED** [review]
LGTM.

Thanks for reaching out with us for confirmation. We also appreciate that you read and followed our contributing guidelines 🙂

**Review by gikf — APPROVED** [review]

**Comment by gikf** [comment]
Hi @RGHANILOO. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Comment by RGHANILOO** [comment]
> Hi @RGHANILOO. Congrats on your first pull request (PR)! 🎉
> 
> Thank you for your contribution to the page! 👍 We are happy to accept these changes and look forward to future contributions. 📝

Thank you, can't wait to get more involved in the future.

**Comment by RGHANILOO** [comment]
> LGTM.
> 
> Thanks for reaching out with us for confirmation. We also appreciate that you read and followed our contributing guidelines 🙂

Thank you , can't wait to get involved more in the future.

**Pull request merged**